### DISCUSSION TITLE
使用textstudio参考文献显示空白

### DISCUSSION THREAD

**Discussion by yuanchude** [community/community#121774] [Programming Help]
### Body

运行没有标红，使用Bibtex编译报错，使用XelaTex又完成编译，但是参考文献什么都不显示，引用也是问号，是哪里出问题了呢？
![微信图片_20240501121942](https://github.com/community/community/assets/111027342/6bda47cf-ed3d-418f-801f-3fa13f066c26)
![微信图片_20240501121938](https://github.com/community/community/assets/111027342/dd52980f-2f32-4b86-8049-4481cff01592)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @yuanchude, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

Please feel encouraged to post again if you are still having this issue!
